modules/ROOT/pages/glossary.adoc

```diff
@@ -0,0 +1,116 @@
+[glossary]
+= Glossary
+
+:description: List of important keywords used in the documentation to better understand the main Bonita concepts.
+
+[caption="Work in Progress"]
+CAUTION: This glossary is a work in progress. It does not include all the terms in Bonita documentation.
+
+== A
+
+Actor filter::
+    Filters users that can access a human task in a process.
+
+Application descriptor::
+    Defines the Living Application: group of pages with a specific theme for a given user profile.
+
+== B
+
+BDM Access Control::
+    Defines rules to protect the access to the Business Data Model (BDM) through REST API calls via a white list mechanism.
+
+Business Data Model::
+    Business Data Model (sometimes abbreviated _BDM_) is the structure of the business data used by your processes.
+
+Bonita Engine::
+    Bonita Engine is at the core, responsible for process execution.
+
+Bonita Platform::
+    Bonita Platform is composed of two components : Bonita Studio and Bonita Runtime. Its purpose is to create, build and run projects.
+
+Bonita Runtime::
+    Bonita Runtime is dedicated to operate automation projects. It is composed of one or more nodes. A Bonita Runtime composed of multiple nodes constitutes a cluster to ensure High Availability and High Performance.
+
+Bonita Studio::
+    Bonita Studio contains all the elements needed to develop and build a project. It is a desktop application based on Eclipse IDE.
+
+Bonita UI Designer::
+    Bonita UI Designer (sometimes abbreviated _UID_) is a development environment used for creating pages and forms that make up a Living Application. It is a web-based tool that is launched from Bonita Studio.
+
+== C
+
+Connector::
+    Automates external services interaction from within processes.
+
+== D
+
+Diagram::
+    BPMN diagram that describes your business processes.
+
+== E
+
+Extensions::
+    Extensions are custom components used to add elements to those provided while others are used to enforce the Bonita platform possibilities. An extension can be an Actor filter, a Connector, a REST API Extension or a Theme. Extensions can be developed outside the Studio with your favorite IDE.
+
+== F
+
+Form::
+    Web interfaces to interact with processes (to instantiate a process and for human tasks).
+
+== L
+
+Layout::
+    Used to define the main frame of a Living Application.
+
+Living Application::
+    A Living Application is a navigation of pages that can be accessed by a Profile. It is created and operated within Bonita Platform.
+
+== O
+
+Organization::
+    The definition of the user base that will use the processes (includes users, roles and groups).
+
+== P
+
+Pages::
+    Web pages used within your applications.
+
+Project::
```

Comment:
It is surprising to have so much details on this. IMO we should have a simple description here, with these deatils moved in the `workspaces-and-repositories` page.